features/works/work_drafts.feature

```diff
@@ -138,3 +138,50 @@ Feature: Work Drafts
           And I follow "Next Chapter"
         Then I should see "Series this work belongs to:"
           And I should see "Aisle 5"
+
+    Scenario: Word count should appear after creating single chapter work, saving as draft, and posting draft from user's drafts page
+      Given I am logged in as "test_user"
+        And I set up the draft "Unicorns are everywhere"
+        And I fill in "content" with "Help there are unicorns everywhere"
+      When I press "Preview"
+        And I press "Save As Draft"
+        And I go to my drafts page
+        And I follow "Post Draft"
+      Then I should be on the work "Unicorns are everywhere"
+        And I should see "Words:5"
+
+    Scenario: Word count should equal all draft chapters' word counts if work isn't posted
+      Given I am logged in as "test_user"
+        And I set up the draft "Unicorns are everywhere"
+        And I fill in "content" with "Help there are unicorns everywhere"
+      When I press "Preview"
+        And I press "Save As Draft"
+      When a chapter is set up for "Unicorns are everywhere"
+        And I press "Preview"
+        And I press "Save As Draft"
+      Then I should see "Words:16"
+      When a chapter is set up for "Unicorns are everywhere"
+        And I press "Preview"
+      When I press "Save As Draft"
+      Then I should see "Words:27"
+
+      Scenario: When posting chapter(s) in unpublished multichapter work, word count should equal posted chapter(s) word count
+      Given I am logged in as "test_user"
+        And I set up the draft "Unicorns are everywhere"
+        And I fill in "content" with "Help there are unicorns everywhere"
+      When I press "Preview"
```

Comment:
This should also probably be an "And", since it's part of the initial set up of the draft (the "Given" steps).